pylintrc

```diff
@@ -0,0 +1,328 @@
+[MASTER]
```

Comment:
No need for both a .pylintrc and a pylintrc file.  The travis.yml uses .pylintrc, but PyLint will find pylintrc by default.